### ISSUE TITLE
Add episode Title on hover

### ISSUE THREAD

**Issue by RensReus** [phiresky/tv-show-ratings#5]

**Comment by phiresky** [comment]
Good Idea. Currently the whole dataset is 4MB. This is something that would increase the size of the dataset by a lot (~50byte per show to maybe ~500byte). Which would probably require rethinking how the data is retrieved or at least changing the chunk size and seeing if that fixes it (since each client fetches the data for all shows eventually, prioritizing the chunk of size 1/256 of the total data that contains the requested show - though this prioritization is not guaranteed to be effective). #6 has a similar problem.
